db/migrate/20150313054707_create_translations.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTranslations < ActiveRecord::Migration
+  def change
+    create_table :translations do |t|
+      t.datetime :time_assigned
+
```

Comment:
No, those are fine and belong in the model, not the schema. The issue is that you don't have a `t.references` field, so your translations have no way to identify the corresponding video and user in the database.